pages/_app.js

```diff
@@ -60,6 +61,8 @@ class OperationCodeApp extends App {
   componentDidMount() {
     /* Analytics */
     // TODO: Leverage master-build-time-only env vars instead of window check
+    loadReCaptcha();
```

Comment:
Oh we won't wanna load this for every page. `_app` is like the middleware for _every_ page's client run-time. It looks like we'll wanna call this function in the page that I asked for via:

```js
useEffect(() => {
  loadReCaptcha();
}, []);
```

Check this out: https://reactjs.org/docs/hooks-effect.html

Reply:
Ok awesome! I remembered seeing that. I just couldn’t recall. Thanks!